src/components/Grid.tsx

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+import { grid, GridProps, space, SpaceProps } from "styled-system";
+
+const Grid = styled.div<GridProps & SpaceProps>`
+  display: grid;
+  ${grid};
+  ${space};
+`;
```

Comment:
Sweeet - let's add `${layout}` too so we can keep the `overflow: scroll` on `FashionEditorial`

Reply:
Yeahhhh